ui/cockpit/src/modules/components/Notifications/Notifications.js

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH
+ * under one or more contributor license agreements. See the NOTICE file
+ * distributed with this work for additional information regarding copyright
+ * ownership. Camunda licenses this file to you under the Apache License,
+ * Version 2.0; you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import React, { useEffect, useState, useCallback } from "react";
+import classNames from "classnames";
+
+import { registerConsumer, unregisterConsumer } from "utils/notifications";
+
+import "./Notifications.scss";
+
+export default function Notifications({ className }) {
+  const [notifications, setNotifications] = useState([]);
+  const [consumerIndex, setConsumerIndex] = useState(null);
+
+  const removeNotification = useCallback(
+    notification => {
+      notifications.splice(notifications.indexOf(notification), 1);
+      setNotifications([...notifications]);
+    },
+    [notifications]
+  );
+
+  useEffect(() => {
+    const consumer = {
+      add: notification => {
+        setNotifications([...notifications, notification]);
+        return true;
+      },
+      remove: removeNotification
+    };
+    setConsumerIndex(registerConsumer(consumer, consumerIndex));
+    return () => {
+      unregisterConsumer(consumer);
+    };
+  }, [consumerIndex, notifications, removeNotification]);
```

Comment:
I'm not really happy with this solution, but could not find anything better. The consumer is dependent on the Notifications (state) and will be recreated when the Notifications change. Therefore, the consumer needs to be re-registered.